backend/app/main.py

```diff
@@ -1083,6 +1085,28 @@ async def get_projects(
     ).all()
     return gerrydb_views
 
+@app.get("/api/document/{document_id}/thumbnail", status_code=status.HTTP_200_OK)
+async def get_thumbnail(
```

Comment:
H: What is the use case you are thinking for this? I think we can just use the CDN path (eg `https://tilesets1.cdn.districtr.org/thumbnails/f3c43f6b-b338-4475-b686-70268b9f16d8`) with a fallback default image for the frontend uses eg:
```
<object
  type="image/png"
  data="https://somedomain.com/image.png"
  width="150"
  height="150"
  aria-label="This image should exist, but alas it does not"
>
  <img src="/path/to/fallback.png" alt="Fallback image" width="150" height="150" />
</object>
```
Would this get called somewhere else?

Reply:
OK since it's accessible through that bucket URL that sounds like what I want to do instead of copying.
I'm thinking ahead to the OpenGraph thumbnail URL, where we want a GET path, a request means we want to generate that thumbnail, and if the S3 image doesn't exist we can redirect to a fallback image.
So I think what I want to do is, redirect to S3 or redirect to the fallback.